Tests/Packet++Test/PacketExamples/nflogPacket.dat

```diff
@@ -0,0 +1 @@
+0200002a080001000000030005000a0000000000080005000000000208000b000000000008000e0000000000410009004500003d021040004011208f0a00020f0a000203a542003500294156c04e0100000100000000000003777777076578616d706c65036e657400000100012f0a31
```

Comment:
To ease review in case of bug/problem etc. all `.dat` files have `pcap` versions. Could you add `pcap` of this packet?